supplier_app/tests/test_model.py

```diff
@@ -134,3 +135,28 @@ def test_company_user_permissions(self):
             company_user_permissions.company,
             company
         )
+
+
+class TestCompanyUniqueToken(TestCase):
+    def setUp(self):
+        self.company = CompanyFactory()
+        self.companyuniquetoken = CompanyUniqueToken(company=self.company)
+
+    def test_company_unique_token_relationship(self):
+        self.assertEqual(
+            self.company,
+            self.companyuniquetoken.company
+        )
+
+    def test_company_token_does_not_repeat(self):
+        tokens = [
+            self.companyuniquetoken._token_generator() for i in range(100)
+        ]
+        self.assertTrue(len(tokens) == len(set(tokens)))
+
+    def test_token_length(self):
+        token = self.companyuniquetoken._token_generator()
+        self.assertEqual(len(token), 64)
+
+    def test_company_unique_token_assing_company_token(self):
+        self.companyuniquetoken.assing_company_token
```

Comment:
No assertion here?

Reply:
Added assertion that a token has been assigned to companyuniquetoken